_docs/howto/deploy/change-gcp-deployment.md

```diff
@@ -1,28 +1,64 @@
 ---
-title: Changing a Deployment
+title: Updating Forseti
 order: 104
 ---
 #  {{ page.title }}
 
-This page describes how to change a Forseti Security deployment. Most changes
-to a deployment are usually to the deployment properties, such as the
-instance type or `src-path` when you want your deployment to run a certain 
-version of Forseti Security. To update your deployment, follow the process below:
+### Update local installation
+If you want to update your local version of Forseti, pull in the latest changes from
+git:
 
-  1. Edit `deploy-forseti.yaml` and update the values you want to change.
+```bash
+$ cd forseti-security
+$ git pull
+```
+
+And then run the python installation:
+
+```bash
+$ python setup.py install
+```
+
+### Change deployment properties
+If you need to change any of the properties for your Forseti deployment, such as the
+`release-version`, follow the process below:
+
+  1. If you **do not** need to update your Forseti version, continue on to step 3.
+     Sync your local git code (where you ran Deployment Manager) so you get the latest 
+     deployment templates. This is important because a newer version of Forseti might 
+     also require an updated startup script on the GCE instance.
+     
+     To get the latest version of Forseti, sync your master branch:
+     
+     ```bash
+     $ git checkout master
+     $ git pull
+     ```
+     
+  1. Check `deploy-forseti.yaml.sample` to see if there are any new properties 
+     that you need to copy over to your previous `deploy-forseti.yaml`.
+  1. Edit `deploy-forseti.yaml` and update the values you want to change. If you 
+     previously deployed the `master` branch, you don't need to change it.
   1. Run the following update command:
 
       ```bash
       $ gcloud deployment-manager deployments update DEPLOYMENT_NAME \
```

Comment:
It's generally best not to include a $ to indicate you're at a command line. The reason is that we want users to be able to copy-paste the relevant commands. If a user pastes "$", the command fails.

Reply:
@mirons-google

Reply:
This is my fault. I realize it breaks the copy paste. I was hoping to change the copy-code button logic to stop at the $. Until that's changed you're correct that this is the better thing to do.

Reply:
I think even if we change the copy button logic, it may still be best to exclude the $ - particularly in cases where it's two separate commands, users will probably manually copy the first, run it, then manually copy the second. Example from above:
```
$ git checkout master
$ git pull
```
For cloud.google.com docs, we have a class that excludes the system prompt from mouse select. Would something like that be possible here?